### PR TITLE
example fix url

### PR DESCRIPTION
**Pull request by ademilter**
## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have a helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have a helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes by running `pnpm build && pnpm lint`
- [ ] The "examples guidelines" are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)

### CODE REVIEW

**Review by steven-tey — DISMISSED** [review]
LGTM!

File: examples/blog-with-comment/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "@types/react-dom": "^18.0.7",
     "autoprefixer": "^10.4.12",
     "postcss": "^8.4.18",
+    "prettier": "^2.7.1",
```
[the remark this answers is not in the document]
Comment (steven-tey) [reply examples/blog-with-comment/package.json:28]: cc: @ademilter

[the remark this answers is not in the document]
Comment (ademilter) [reply examples/blog-with-comment/package.json]: @steven-tey ✅

**Review by steven-tey — APPROVED** [review]

File: examples/blog-with-comment/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (steven-tey) [reply examples/blog-with-comment/package.json]: thank you! merged!

**Pull request merged**